toolbox.mk

```diff
@@ -40,6 +50,8 @@ banner = \
 # Macros for executing running a command in the toolbox container.
 _toolbox = \
 	docker run --rm $2 \
+		-e HOST_ROOT_DIR \
+		-e HOST_HOME_DIR \
```

Comment:
Can I suggest we don't export those variables at the top of this file and instead have:

```
	docker run --rm $2 \
		-e HOST_WORK_DIR=$(shell pwd) \
		-e HOST_HOME_DIR=$(HOME) \
```

This is simpler and I don't think we want the directory that hosts the `toolbox.mk` file (even though it'll almost always be the same as the `Makefile` directory which will be the same as the CWD). But if we're going to choose one, we should choose the CWD because that will be the where the `target/` directory is which will have the build output we're interested in.

I also renamed `HOST_ROOT_DIR` to `HOST_WORK_DIR` because I think "root" is a bit misleading.